python/lsst/meas/base/forcedPhotCoadd.py

```diff
@@ -121,9 +132,21 @@ class ForcedPhotCoaddConfig(pipeBase.PipelineTaskConfig,
             "Must have IDs that match those of the reference catalog."
             "If None, Footprints will be generated by transforming the reference Footprints.",
         dtype=str,
-        default="meas",
+        default="scarletDeblendModels",
         optional=True
     )
+    doConserveFlux = lsst.pex.config.Field(
+        dtype=bool,
+        default=True,
+        doc="Whether to use the deblender models as templates to re-distribute the flux "
+            "from the 'exposure' (True), or to perform measurements on the deblender model footprints. "
+            "If footprintDatasetName != 'scarletDeblendModels' then this field is ignored.")
+    doStripFootprints = lsst.pex.config.Field(
+        dtype=bool,
+        default=True,
+        doc="Whether to strip footprints from the output catalog before "
+            "saving to disk. "
+            "This is usually done when using scarlet models to save disk space.")
```

Comment:
If you keep the current behavior:

```suggestion
            "This is usually done when using scarlet models to save disk space. "
            "If footprintDatasetName != 'scarletDeblendModels' then this field is ignored.")
```

But I actually think this is a good option to have even when not using Scarlet models, and in fact it should be the default (though that might cause too much trouble downstream to do on this ticket) - these `HeavyFootprints` are always exactly the same as those stored in `deepCoadd_meas`, right?

Reply:
Yes. I checked out a `meas` and `forced_src` catalogs for the same patch and every heavy footprint was the same, which are the same as the ones in the `deblendedFlux` catalog when using either scarlet or the SDSS-like deblender catalog. So we shouldn't be storing them in `meas` or `forced_src` regardless of the deblender used, and I can make that change in this ticket for coadds.